app-misc/task/task-2.6.2.ebuild

```diff
@@ -68,3 +68,7 @@ src_install() {
 	exeinto "/usr/share/${PN}/scripts"
 	doexe scripts/add-ons/*
 }
+
+pkg_postinst() {
+	einfo "To use XDG directories, please edit your .taskrc after issuing the first task command"
```

Comment:
Could you hint what needs to be done exactly? Also maybe use `elog` so the message gets saved, and is more verbose.
You can split it into multiple lines.